miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/PartitionQCTypeDto.java

```diff
@@ -56,11 +56,11 @@ public String getDetailedLabel() {
     sb.append(getDescription());
     sb.append(" (order ");
     if (!isOrderFulfilled()) sb.append("not ");
-    sb.append("fulfilled, analysis ");
+    sb.append("fulfilled, ");
     if (isAnalysisSkipped()) {
```

Comment:
It looks like this is getting renamed everywhere except this method name?

Reply:
The field name has not been changed - only the labels